skimage/feature/tests/test_orb.py

```diff
@@ -120,3 +122,10 @@ def test_no_descriptors_extracted_orb():
     detector_extractor = ORB()
     with testing.raises(RuntimeError):
         detector_extractor.detect_and_extract(img)
+
+
+@pytest.mark.parametrize('dtype', ['float32', 'float64', 'uint8', 'int'])
```

Comment:
Int64?

Reply:
:+1: I am trying to find a more elaborated test

Reply:
let me still merge this now you're welcome to submit an other PR with a more complete test of course.